xbmc/interfaces/json-rpc/AudioLibrary.cpp

```diff
@@ -41,6 +41,25 @@ using namespace JSONRPC;
 using namespace XFILE;
 using namespace KODI::MESSAGING;
 
+JSONRPC_STATUS CAudioLibrary::GetProperties(const std::string &method, ITransportLayer *transport, IClient *client, const CVariant &parameterObject, CVariant &result)
+{
+  CVariant properties = CVariant(CVariant::VariantTypeObject);
+  for (unsigned int index = 0; index < parameterObject["properties"].size(); index++)
```

Comment:
Please use iterators with `CVariant::begin_array()` and `CVariant::end_array()`.